src/cargo/core/features.rs

```diff
@@ -353,6 +356,7 @@ pub struct CliUnstable {
     pub crate_versions: bool,
     pub separate_nightlies: bool,
     pub multitarget: bool,
+    pub strip: bool,
```

Comment:
I think this can be removed since this is a manifest flag instead of a `-Z` flag